### PR TITLE
Remove rank from contest voting

### PR DESCRIPTION
**Pull request by lizchow**
Remove the concept of rank from contest voting. The scoring for voting is now 1(worst) to 10 (best). 

Should be tested with the frontend PR.

### CODE REVIEW

File: lib/cadet_web/views/assessments_helpers.ex [hunk 1]
```diff
@@ -140,7 +140,7 @@ defmodule CadetWeb.AssessmentsHelpers do
         answer: :answer,
         student_name: :student_name
       }),
-      "score",
+      "relative_score",
```
Comment (YaleChen299) [inline lib/cadet_web/views/assessments_helpers.ex:143]: will this be confusing with the score? Is this relative score the computed final score on the leaderboard? Maybe we can name it `final_score` or something along those line?

**Review by YaleChen299 — COMMENTED** [review]
Generally looks good to me! Just one minor issue on naming that may affect the frontend side as well.

File: lib/cadet_web/views/assessments_helpers.ex (hunk 1 above)
Comment (lizchow) [reply lib/cadet_web/views/assessments_helpers.ex]: ok

**Comment by YaleChen299** [comment]
percentage dropped due to removed parts, no new uncovered lines.

**Pull request merged**